Dockerfile

```diff
@@ -22,10 +22,12 @@ RUN apt-get update && \
 
 # Stage: build assets
 FROM base as build
-ARG BUILD_NUMBER
-ARG GIT_REF
 
-RUN apt-get install -y make python g++
+ARG BUILD_NUMBER=1_0_0
+ARG GIT_REF=not-available
+
```

Comment:
Do these get overwritten with the proper values in the second build stage? Just checking that the dps-monitor will retain the actual Git ref and build version

Reply:
Yes seems to be ok, this is copied direct from the template repo

<img width="656" alt="Screenshot 2022-03-24 at 18 54 37" src="https://user-images.githubusercontent.com/30229564/159979755-87e279bb-918a-4ce3-9d3a-004cecf758a4.png">